fiat/prices.go

```diff
@@ -100,19 +101,21 @@ func getQueryableDuration(requests []*PriceRequest) (time.Time, time.Time) {
 // msatToUSD converts a msat amount to usd. Note that this function coverts
 // values to Bitcoin values, then gets the fiat price for that BTC value. If
 // an amount < 1000 msat is given, a zero amount will be returned.
-func msatToUSD(price float64, amt lnwire.MilliSatoshi) float64 {
-	btcBal := amt.ToBTC()
-	return price * btcBal
+func msatToUSD(price decimal.Decimal, amt lnwire.MilliSatoshi) decimal.Decimal {
+	msatDecimal := decimal.NewFromFloat(amt.ToBTC())
```

Comment:
I wonder if it'd be better to write this as:
```
pricePerMilliSatoshi = price.Div(decimal.NewFromInt(100000000000))
return amt.Mul(pricePerMilliSatoshi)
```

Reply:
Yeah with a comment this is more readable 👍

Reply:
More like more precise as we avoid the float conversion.